test/src/test/java/com/buschmais/jqassistant/commandline/test/AbstractCLIIT.java

```diff
@@ -30,12 +30,12 @@
  */
 public abstract class AbstractCLIIT {
 
-    public enum NeoVersion {
-        NEO4JV3("neo4jv3");
+    public enum Neo4jVersion {
```

Comment:
Do we actually need this enum if there is only one option anyway?

Reply:
Removed, furthermore removed neo4jv4 suffix from CLI distribution